Tests/ClientTest.php

```diff
@@ -3,75 +3,81 @@
 namespace TruckersMP\Tests\API;
 
 use Carbon\Carbon;
-use TruckersMP\API\APIClient;
-use TruckersMP\Types\Ban;
-use TruckersMP\Types\Bans;
-use TruckersMP\Types\Player;
+use PHPUnit\Framework\TestCase;
+use TruckersMP\Client;
+use TruckersMP\Models\BanModel;
+use TruckersMP\Models\BansModel;
+use TruckersMP\Models\PlayerModel;
 
-class ClientTest extends \PHPUnit_Framework_TestCase
+class ClientTest extends TestCase
 {
-    private $testAccount = 585204;
+    /**
+     * @var int
+     */
+    protected $testAccount = 162950;
```

Comment:
I see what you did there but this is not reliable at all. We might issue a ban on our test account. Also, there is no check whethever bans are hidden on account.

Reply:
I'll revert it to the test account and so sanity checks on the data. I agree that this is not the best idea. I'll implement the handling for no bans too and test accordingly.